cachecontrol.go

```diff
@@ -29,15 +29,15 @@ func ParseCacheControl(input string) (CacheControl, error) {
 		var token string
 		switch input[pos] {
 		case '"':
-			token = readString(input, pos+1, "\"")
+			token = readUntil(input, pos+1, "\"")
 			pos += len(token) + 1
 		case ',', '\n', '\r', ' ', '\t':
 			continue
 		case '=':
 			isValue = true
 			continue
 		default:
-			token = readString(input, pos, "\"\n\t\r ,=")
+			token = readUntil(input, pos, "\"\n\t\r ,=")
```

Comment:
Ah yup, you are totally correct. I'd always rather use a built-in over my own code